src/ServiceManagement/Azure.psd1

```diff
@@ -51,7 +51,7 @@ CLRVersion = '4.0'
 # ProcessorArchitecture = ''
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @(@{ModuleName = 'Azure.Storage'; ModuleVersion = '4.5.0'; })
+RequiredModules = @(@{ModuleName = 'Azure.Storage'; ModuleVersion = '4.5.1'; })
```

Comment:
Please do not increase the module version numbers. We handle version number incrementation when we create releases.